recipes/default.rb

```diff
@@ -27,27 +27,18 @@
 chef_ingredient 'chef-server' do
   version node['chef-server']['version']
   package_source node['chef-server']['package_source']
+  config <<-EOS
+topology "standalone"
```

Comment:
Perhaps we should make a `node['chef-server']['topology']` attribute, and default to `standalone`? That would give us some flexibility for reuse.